framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/widget/Widget.module.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../styles/constants';
+
+.root {
+	display: block;
+	margin: 10px 0;
+}
+
+.header {
+	color: inherit;
+	margin-bottom: 15px;
+	font-family: 'Roboto';
+	font-style: normal;
+	font-weight: 600;
+	font-size: 20px;
+	line-height: 130%;
+	letter-spacing: 0.230769px;
+	color: $color-white;
+}
+
+.body {
+	background: $color-background-darker;
+	border-radius: 6px;
+	color: $color-white;
+	padding: 20px;
+	width: 100%;
+	box-sizing: border-box;
+	display: block;
+}
+
+.widget-body-scrollbar {
```

Comment:
I think scrollbar has bit more style in design

Reply:
Its fixed for Chrome but didn't compatible with Firefox.